pkg/packaging/detectLauncherVersion_test.go

```diff
@@ -52,5 +54,37 @@ func TestFormatVersion(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, tt.out, version)
 	}
+}
+
+func TestLauncherLocation(t *testing.T) {
+	t.Parallel()
+
+	pDarwin := &PackageOptions{target: Target{Platform: Darwin}}
+
+	// First, test that if the app bundle doesn't exist, we fall back to the top-level binary
+	require.Equal(t, filepath.Join("some", "dir", "launcher"), pDarwin.launcherLocation(filepath.Join("some", "dir")))
+
+	// Create a temp directory with an app bundle in it
+	tmpDir := t.TempDir()
+	require.NoError(t, os.MkdirAll(filepath.Join(tmpDir, "Kolide.app", "Contents", "MacOS"), 0755))
+	baseDir := filepath.Join(tmpDir, "Kolide.app", "Contents", "MacOS")
+	expectedLauncherBinaryPath := filepath.Join(baseDir, "launcher")
+	f, err := os.Create(expectedLauncherBinaryPath)
+	if err != nil {
+		t.Errorf("could not create temp file for test: %v", err)
+		t.FailNow()
+	}
+	defer require.NoError(t, f.Close())
+	defer require.NoError(t, os.Remove(expectedLauncherBinaryPath))
```

Comment:
I wouldn't wrap these in `require.NoError`

Reply:
Oh, fair, makes sense